### PR TITLE
IUV Militiaman (new vox ship)

### PR DESCRIPTION
**Pull request by ghost**
A new vox away site because it was requested and I am a chimpanzee. I'm not touching the old one unless told to. I also renamed `/material/voxalloy` from its stupid generic name to something mildly Vox-related. This is a draft. If you even try and tell me tests are failing while this PR is colored gray, I am going to enter a martial trance. **Don't run CI yet.**

As for lore, it's a decommissioned human-constructed monitor reutilized by the Vox. Yes, I know the turfs will need to be changed. I also know I screwed up the name in the initial commit.

### CODE REVIEW

File: maps/away/vox/vox_overmap.dm [hunk 1]
```diff
@@ -0,0 +1,60 @@
+//The actual overmap object
+/obj/effect/overmap/visitable/ship/iuv_militiaman
+	name = "IUV Militiaman"
+	desc = "An anomalously FTL-capable vessel bearing unknown markings. Its designation \
+	is marked as already having been destroyed. There are consistent life signs onboard. \
```
Comment (ghost) [inline maps/away/vox/vox_overmap.dm:5]:
```suggestion
	is marked as already having been decommissioned. There are consistent life signs onboard. \
```

File: maps/away/vox/vox_overmap.dm [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+/decl/ship_contact_class/militiaman
+	class_short = "TENNESSEE" //I'll haunt this code forever
+	class_long = "Tennessee-Class Monitor"
+	max_ship_mass = 7500
+
+//The shuttle
+/obj/effect/overmap/visitable/ship/landable/iuv_diplomat
+	name = "IUV Diplomat"
+	desc = "A small-sized medium-ranged shuttle bearing unknown markings. Its \
+	designation is marked as already having been destroyed."
```
Comment (ghost) [inline maps/away/vox/vox_overmap.dm:35]:
```suggestion
	designation is marked as already having been decommissioned."
```

File: maps/away/vox/vox_overmap.dm [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+/obj/effect/overmap/visitable/ship/iuv_militiaman
+	name = "IUV Militiaman"
+	desc = "An anomalously FTL-capable vessel bearing unknown markings. Its designation \
+	is marked as already having been destroyed. There are consistent life signs onboard. \
+	Various armaments are present on the vessel."
+
+	identification_difficulty = 150 //It's a hostile alien vessel
+	contact_class = /decl/ship_contact_class/militiaman
+
+	vessel_mass = 7500 //The Bearcat is 4000 for reference
```
Comment (ghost) [inline maps/away/vox/vox_overmap.dm:11]: Up for debate.

**Comment by Anton-Kr** [comment]
Please add to away sites testing.

**Comment by Armyguy** [comment]
> 

Its not testing as its still looking for the Voxship Voxship it seems like. Or at least I think?, I don't know 100% how CI works

File: maps/away/vox/vox_areas.dm [hunk 4]
```diff
@@ -0,0 +1,11 @@
+/area/vox
+	icon = 'vox_areas.dmi'
+	req_access = list(access_vox)
```
Comment (ghost) [inline maps/away/vox/vox_areas.dm:3]: Need to define this access.

File: maps/away/vox/vox_overmap.dm [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+	vessel_size = SHIP_SIZE_MEDIUM
```
Comment (ghost) [inline maps/away/vox/vox_overmap.dm:12]:
```suggestion
	vessel_size = SHIP_SIZE_SMALL
```

[Hmm yes, size variation of tiny, small, and large.](https://github.com/BoHBranch/BoH-Bay/blob/d4febd53b007a843249a34da0f9254e66c5c364e/code/__defines/overmap.dm#L2)

**Comment by ghost** [comment]
> Please add to away sites testing.

I will add it when it's functional enough not to immediately fail.

> Its not testing as its still looking for the Voxship Voxship it seems like. Or at least I think?, I don't know 100% how CI works

What Anton said, I just haven't added it to the testing list yet. It's considered a seperate away site from the `away/voxship` in terms of actual code.

File: maps/away/vox/vox_overmap.dm [hunk 6]
```diff
@@ -0,0 +1,60 @@
...
+	contact_class = /decl/ship_contact_class/militiaman
+
+	shuttle = "IUV Diplomat"
+	fore_dir = WEST
+	vessel_size = SHIP_SIZE_SMALL
+
+/decl/ship_contact_class/diplomat
+	class_short = "PACER"
+	class_long = "Pacer-Class Shuttle"
+	max_ship_mass = 7500
```
Comment (ghost) [inline maps/away/vox/vox_overmap.dm:47]: Fix this to whatever value is proper for the shuttle, not the entire monitor.

File: maps/away/vox/vox_overmap.dm [hunk 7]
```diff
@@ -0,0 +1,60 @@
...
+
+//The shuttle
+/obj/effect/overmap/visitable/ship/landable/iuv_diplomat
+	name = "IUV Diplomat"
+	desc = "A small-sized medium-ranged shuttle bearing unknown markings. Its \
+	designation is marked as already having been destroyed."
+
+	identification_difficulty = 150
+	contact_class = /decl/ship_contact_class/militiaman
```
Comment (ghost) [inline maps/away/vox/vox_overmap.dm:38]:
```suggestion
	contact_class = /decl/ship_contact_class/diplomat
```

**Comment by ghost** [comment]
I am going to hold this off because we have a semi-functioning Vox away site and I see more pressing matters.

**Pull request closed without merging**